.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    name: "Validate with Node v${{ matrix.node-version }}"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ["16.17.0", "18.12.0"]
```

Comment:
wonder if it's a good idea to also add `lts` here.

Reply:
Great idea - it looks like they support `current` so I went with that, and loosened the node versions to test against major versions only. See a036173